psi4/src/psi4/libscf_solver/hf.cc

```diff
@@ -153,18 +153,18 @@ void HF::common_init()
         if(old_pg){
             // This is one of a series of displacements;  check the dimension against the parent point group
             size_t full_nirreps = old_pg->char_table().nirrep();
-            if(options_["DOCC"].size() != full_nirreps)
-                throw PSIEXCEPTION("Input DOCC array has the wrong dimensions");
```

Comment:
This is still an important check. Other than that, I think everything looks good.